code/game/jobs/job/engineering.dm

```diff
@@ -182,3 +182,33 @@
 	satchel = /obj/item/storage/backpack/satchel_eng
 	dufflebag = /obj/item/storage/backpack/duffel/eng
 	messengerbag = /obj/item/storage/backpack/messenger/engi
+
+/datum/job/officer/eng
+	title = "Engineering Officer"
+	flag = ENGISEC
+	department = "Engineering"
+	department_flag = ENGSEC
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the chief engineer and head of security"
+	selection_color = "#dbba35"
+	economic_modifier = 4
+
+	minimum_character_age = 18
+
+	access = list(access_security, access_brig, access_sec_doors, access_eva, access_external_airlocks, access_weapons, access_engine, access_maint_tunnels, access_construction)
```

Comment:
Should never have access to the engine, that is only useful if they want a quick death.

Reply:
The "access_engine" datum refers to just general "Engineering" access, the actual Engine Room access datum is "access_engine_equip", which I removed from them already.